### PR TITLE
feat: migrated app to NuxtBridge 📦 

### LINKED ISSUE #1418: Add Vite to main so we can be at awesome-vite
Because, why not?
image

Seems we can use bridge right away https://v3.nuxtjs.org/getting-started/bridge/

## Ref 
- https://github.com/vitejs/awesome-vite
- https://github.com/nuxt/vite -- seems Vite is part of NuxtBridge 
- https://v3.nuxtjs.org/getting-started/bridge/

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1418, #2675
- [x] migrated app to Nuxt Bridge, nitro is disabled in production

### Before submitting Pull Request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by kkukelka** [comment]
hmm somehow deploy fails 😨

**Comment by preschian** [comment]
need to run `pnpm install` first and then commit the lockfiles. recently, pnpm PR got merged

Screen Shot 2022-03-26 at 12 23 07 AM

**Comment by kkukelka** [comment]
> need to run `pnpm install` first and then commit the lockfiles. recently, pnpm PR got merged
> 
> Screen Shot 2022-03-26 at 12 23 07 AM

oh wow. That completely passed me by. 🙇

**Comment by yangwao** [comment]
Do I see right that pnpm took 27s and total whole build 28s? :D 

image

**Comment by yangwao** [comment]
I guess webpack could be removed now?

**Comment by kkukelka** [comment]
> I guess webpack could be removed now?
> 
> related as well #2675

I'm actually unsure how this will effect webpack extension is our config
```
extend(config) {
      // add markdown loader
      config.module.rules.push({
        test: /\.md$/,
        use: 'raw-loader',
      })

      config.module.rules.push({
        test: /\.js$/,
        loader: require.resolve('@open-wc/webpack-import-meta-loader'),
      })
      // config.resolve.alias.vue = 'vue/dist/vue.common' //https://github.com/nuxt/nuxt.js/issues/1142#issuecomment-317272538
      config.resolve.alias['vue$'] = 'vue/dist/vue.esm.js'
      config.node = {
        fs: 'empty',
      }
    }
```
but i will give it a try later

**Comment by kkukelka** [comment]
@yangwao i guess it worked 👀

**Comment by yangwao** [comment]
okay so if it's new build takes like 4m ~ but once it's from cache it's ~30s, nice!

**Comment by yangwao** [comment]
Waiting for acknolwedgment

image

File: nuxt.config.js [hunk 1]
```diff
@@ -278,20 +290,21 @@ export default {
...
-      aggregateTimeout: 300,
-      poll: 1000,
-    },
-  },
+  // watchers: {
+  //   webpack: {
+  //     aggregateTimeout: 300,
+  //     poll: 1000,
+  //   },
+  // },
```
Comment (vikiival) [inline nuxt.config.js:307]:
```suggestion
```

File: nuxt.config.js [hunk 2]
```diff
@@ -278,20 +290,21 @@ export default {
         test: /\.js$/,
         loader: require.resolve('@open-wc/webpack-import-meta-loader'),
       })
-      config.resolve.alias.vue = 'vue/dist/vue.common' //https://github.com/nuxt/nuxt.js/issues/1142#issuecomment-317272538
+      // config.resolve.alias.vue = 'vue/dist/vue.common' //https://github.com/nuxt/nuxt.js/issues/1142#issuecomment-317272538
```
Comment (vikiival) [inline nuxt.config.js:293]:
```suggestion
```

**Review by vikiival — APPROVED** [review]
@yangwao or @kkukelka pls delete the comment in the config :)

**Comment by yangwao** [comment]
😱 
10:50:00 AM: Finished processing build request in 6m17.745995787s

**Comment by yangwao** [comment]
cloudflare says oook 
image

**Comment by yangwao** [comment]
if I'll be living on Mars, 1 approval is enough 

image

**Comment by yangwao** [comment]
Now it's time to put KodaDot on **awesome-vite** list 
Hence @kkukelka done this amazing heavy lifting job, he got pre reserved credits for fame 😍

📺   https://github.com/vitejs/awesome-vite

**Comment by yangwao** [comment]
sending extra buck for pushing extra weight over the weekend @kkukelka 😘
pay 400 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $400 @ 167.42 USD/KSM ~ 2.389 $KSM
🧗 [EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW](https://kusama.subscan.io/account/EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)
🔗 [0x9703cb6682700b1c997cb40d0c9da3fb564b5bd87cca95de51b7dbdf8f2d641b](https://kusama.subscan.io/extrinsic/0x9703cb6682700b1c997cb40d0c9da3fb564b5bd87cca95de51b7dbdf8f2d641b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**